### ISSUE TITLE
2FA support?

### ISSUE THREAD

**Issue by theotherguy2175** [shauntarves/wyze-sdk#107]
My account is setup with 2FA will this cause an issue to use this?

**Comment by shauntarves** [comment]
@theotherguy2175 not at all. This library has full support for 2FA. You can either create a client and enter a 2FA **code** manually when prompted, or you can provide you totp **key** to the client creation and the 2FA code will be automatically input for you:

```
client = Client(email=os.environ['WYZE_EMAIL'], password=os.environ['WYZE_PASSWORD'], totp_key=XXXXX)
```
